web_listview_range_select/static/src/xml/web_listview_range_select.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates xml:space="preserve">
+    <t t-inherit="web.ListRenderer.RecordRow" t-inherit-mode="extension">
+        <xpath expr="//tr[@class='o_data_row']/td/CheckBox" position="replace">
```

Comment:
Instead of replacing the `CheckBox`, can't you just add the `t-on-click.capture` attribute? This way any eventual attributes customization will be preserved.

Reply:
@nileshsheliya please look at this.

Reply:
@tarteo Yes, I have updated code accordingly.